portfolio/src/main/java/com/google/sps/data/Comments.java

```diff
@@ -0,0 +1,72 @@
+package com.google.sps.data;
+
+import java.util.ArrayList; 
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
+import com.google.gson.Gson;
+
+public class Comments {
+    
+    private Entity taskEntity;
+    private DatastoreService datastore;
+    private static Comments comments;
+    private Query query;
+    private Gson gson;
+
+    private Comments() {
+        this.taskEntity = new Entity("Comment"); 
+        datastore = DatastoreServiceFactory.getDatastoreService();
+        query = new Query("Comment");
+        gson = new Gson();
+    }
+
+
+    public static Comments getInstance() {
+
+        if(comments != null) {
+            return comments;
+        }
+
+        comments = new Comments();
+
+        return comments;
+    }
+
+
+    public Comment create(String text) {
+
+        Comment comment = new Comment(text);
+
+        taskEntity.setProperty("text",  comment.getText());
+        taskEntity.setProperty("timestamp", comment.getTimeStamp());
+        datastore.put(taskEntity);
+
+        return comment;
```

Comment:
Minor: Every comment that's added will have the same ID. It doesn't matter right now, but this could eventually lead to bugs in the client if multiple new comments are stored in memory and share the same identifier (e.g. "edit comment" functionality would use the wrong identifier).

The `.put(..)` method returns a Key that has the missing info.